components/aws/sagemaker/common/_utils.py

```diff
@@ -517,7 +559,17 @@ def create_endpoint_config_request(args):
 
     return request
 
+def get_endpoint_config_name(client, args):
+    endpoint_config_name = args['endpoint_config_name']
```

Comment:
looks like we have two places where we are trying to generate endpoint_config_name, one is line 572 in this PR and other is [here](https://github.com/kubeflow/pipelines/blob/master/components/aws/sagemaker/common/_utils.py#L488)

Can we remove one/merge these? Also need to handle the case where user hasnt passed the endpoint config name (we dont want to call endpoint_config_name_exists() for a None value)